openshift/example_config/rhsm-conduit.conf

```diff
@@ -1 +1,4 @@
-hostInventoryServiceUrl=localhost
+hostInventoryServiceHostname=localhost
+hostInventoryServicePort=8080
+hostInventoryServiceBaseUrl=/r/insights/platform/inventory/api/v1
+hostInventoryServiceSecure=false
```

Comment:
Why the added complexity? If all we are doing with this is constructing a URL, I'd rather have just one config attribute to specify.

Reply:
I see that putting in a value of "localhost" in the example was misleading... Sorry about that.

Reply:
Fair points. I'll revert this change.

Reply:
REVERTED - Will be included in next commit.